### PR TITLE
Add changes to readme file

### PR DESCRIPTION
**Pull request by jcalderin**

### CODE REVIEW

File: test/acceptance/README.md [hunk 1]
```diff
@@ -75,3 +77,41 @@ used to deploy the environment, needs it.
 
 There is a tag for each feature file
 
+```
```
Comment (fgalan) [inline test/acceptance/README.md]: How do you plan to keep this list updated? Carefull review in each PR adding new acceptance test?

File: test/acceptance/integration/administration/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__author__ = 'Jon'
```
Comment (fgalan) [inline test/acceptance/integration/administration/__init__.py]: License header missing.

File: test/acceptance/tools/deploy_pep.py [hunk 3]
```diff
@@ -36,7 +36,7 @@ def set_variables_config(host_proxied_ip, host_proxied_port, port_listening,
                          log_level,
                          plug_in, plug_in_extract_action,
                          bypass_activation='false', bypass_id='',
-                         cache_users='1000', cache_projects='1000', cache_roles='60'):
+                         cache_users='1000', cache_projects='1000', cache_roles='60', administration_port='11211'):
```
Comment (fgalan) [inline test/acceptance/tools/deploy_pep.py]: Maybe administration_port should be unhardwired.

Comment (fgalan) [inline test/acceptance/tools/deploy_pep.py]: I mean, unhardwire it to the properties.py file (in the case it makes sense).

File: test/acceptance/tools/resources/config_template.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ config.resource = {
         /**
          * Administration port for the proxy.
          */
-        adminPort: 11211
+        adminPort: {{administration_port}}
```
Comment (fgalan) [inline test/acceptance/tools/resources/config_template.js]: mmm... this seems unhardwiring... so probably my previous comment doesn't make sense (in a second look, it seems that in that place you are definigint the default parameter in the function signature).

File: test/acceptance/README.md (hunk 1 above)
Comment (jcalderin) [inline test/acceptance/README.md]: I have an auto mated script to maintain this

NTC

File: test/acceptance/integration/administration/__init__.py (hunk 2 above)
Comment (jcalderin) [inline test/acceptance/integration/administration/__init__.py]: Fixed in 2b5b781

File: test/acceptance/tools/resources/config_template.js (hunk 4 above)
Comment (jcalderin) [inline test/acceptance/tools/resources/config_template.js]: Yes, the parameter is set in the pep config by the properties test file. In the function there is a default value (if you want, you can omit the value set properties, and use the default one)

NTC

File: test/acceptance/tools/deploy_pep.py (hunk 3 above)
Comment (jcalderin) [inline test/acceptance/tools/deploy_pep.py]: Answered in https://github.com/telefonicaid/fiware-orion-pep/pull/189#discussion_r25626459

NTC

**Comment by fgalan** [comment]
LGTM (passing the ball to @iariasleon for final LGTM + merge + delete branch)

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**